Automattic/jetpack
# My Jetpack: Add Jetpack Protect

This PR is the first step in introducing Jetpack Protect standalone plugin to My Jetpack.

#### Changes proposed in this Pull Request:

* Adds the list of installed plugins to the default state.
* Adds a `hasStandalonePluginInstalled()` selector.
* Renames `ScanCard` to `ScanAndProtectCard`, and uses the `protect` slug whenever the site already has a standalone plugin installed.

#### Other information:


#### Jetpack product discussion



1201069996155224-as-1203332781722597

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Visit My Jetpack on a site with only Jetpack installed, and validate the "Scan" card is shown.
* Install a standalone plugin other than Protect, such as Backup. Validate that the "Protect" card is shown in My Jetpack.
* Install Protect via the card in My Jetpack.
* Validate the installation works, Protect is available, and can be managed via My Jetpack.
